apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -360,6 +362,16 @@ function buildPatDeviceConnectionStatusObservable() {
     return timer(0, devicePollingIntervalMs).pipe(
       switchMap(async () => {
         try {
+          // We only try to connect a PAT device if we are currently in a voter session.
+          const authStatus = await auth.getAuthStatus(
+            constructAuthMachineState(workspace)
+          );
```

Comment:
Is it possible to accomplish this with a [guarded transition](https://stately.ai/docs/xstate-v4/xstate/transitions-and-choices/guards) where we handle `PAT_DEVICE_(DIS)CONNECTED` ([src](https://github.com/votingworks/vxsuite/blob/main/apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts#L490-L497))?

This would mean we don't have to make the observable aware of `auth` and keeps the division of responsibility between the observable as an event creator and state machine as an event consumer.